src/app/zwiazek/kalendarz/_components/CalendarMain.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { CalendarHeader } from "@/app/zwiazek/kalendarz/_components/CalendarHeader";
+import { CalendarTable } from "@/app/zwiazek/kalendarz/_components/CalendarTable";
+import { addMonth } from "@/app/zwiazek/kalendarz/_components/Dates/Dates";
+import { cardsPlaceholder } from "@/app/zwiazek/kalendarz/_components/Events/List";
+import { Posts } from "@/app/zwiazek/kalendarz/_components/Events/Posts";
+import { CalendarCard } from "@/components/shared/CalendarCard/CalendarCard";
+import { useState } from "react";
+
+export const CalendarMain = () => {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [postsToShow, setPostsToShow] = useState(3);
+
+  const handleNextMonth = () => {
+    setCurrentDate(addMonth(currentDate, 1));
+    setPostsToShow(3);
+  };
+
+  const handlePrevMonth = () => {
+    setCurrentDate(addMonth(currentDate, -1));
+    setPostsToShow(3);
+  };
+
+  const eventForToday = getEventForToday();
+  console.log(eventForToday);
+
+  return (
+    <>
+      <section className="flex w-full flex-col items-center gap-10 self-center bg-greenMain py-9 tablet:justify-center tablet:gap-52 tablet:px-10 desktop:flex-row">
+        <CalendarTable
+          currentDate={currentDate}
+          prevMonth={handlePrevMonth}
+          nextMonth={handleNextMonth}
+        />
+        <CalendarCard
+          date={eventForToday?.date.split(".")[2]}
```

Comment:
I believe it should just use the `currentDate`.

The `getEventForToday()` function could be used within `<CalendarCard />` component making the `description` prop obsolete - although having a possibility to configure the card via props is always handy.

Reply:
Move it inside `<CalendarCard />`